idea/src/org/jetbrains/kotlin/idea/intentions/AddBracesIntention.kt

```diff
@@ -68,7 +66,7 @@ class AddBracesIntention : SelfTargetingIntention<KtElement>(KtElement::class.ja
         }
 
         // Check for single line if expression
-        if (element is KtIfExpression && expression.isInSingleLine() && element.`else` == null) {
+        if (element is KtIfExpression && element.`else` == null) {
```

Comment:
Please change (or delete) the comment above, because after your change it becomes incorrect.